keyboards/keychron/c3_pro/ansi/red/info.json

```
@@ -0,0 +1,137 @@
+{
+    "usb": {
+        "pid": "0x0430",
+        "device_version": "1.0.1"
+    },
+    "features": {
+        "led_matrix": true
+    },
+    "led_matrix": {
+        "driver": "CKLED2001",
```

Should fix CI build error
```suggestion
        "driver": "ckled2001",
```

---

Thanks!

---

Hi, "ckled2001" caused the compilation to fail, so i changed it back.

---

It's likely because the branch for this PR isn't synced with upstream QMK, looking at the [`common_features.mk`](https://github.com/Keychron/qmk_firmware/blob/keychron_c3_pro/builddefs/common_features.mk#L355) file for example.

---

Hi, exactly, thanks!